### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by dstendardi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            "email": "sven.eisenschmidt@gmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.2",
+        "silex/silex": ">=1.0.0-dev"
+    },
+    "suggest": {
+        "predis/predis": "0.7.0-dev",
+        "twig/twig": "1.5.0-dev"
```
Comment (stof) [inline composer.json]: many suggestions are missing as there is a bunch of other potential dependencies depending of the extension used

**Comment by dstendardi** [comment]
I didn't put  libraries with no version (like doctrine-odm & assetics) because i thought it was required.

**Comment by stof** [comment]
it is better to put an explicit version but otherwise you can still use `"*"`. Recommend and suggest should allow to know all optional dependencies of the library

**Comment by sveneisenschmidt** [comment]
@dstendardi Can you make @stof suggestions and I happy to merge thi spull request.

**Comment by dstendardi** [comment]
ok, sorry for the delay, i'll do it this evening

**Comment by dstendardi** [comment]
I have added missing composer packages to the composer.json. 
However, I really think it's a bad practice to allow dependencies without version as it can break the build. 
Also, the --install-suggests switch seems to be broke (nothing happens when i use it with a fresh install)
Let me know if something is wrong.

**Comment by stof** [comment]
@dstendardi it is not broken. It is simply not implemented yet in the solver.

**Comment by igorw** [comment]
I think it would really be better to split this up into many repos. Then the exact dependency could be specified on the extension level. But I suppose it is somewhat unpractical in this case.

**Comment by stof** [comment]
splitting into several repos also allows getting only the extension you need. For a micro-framework, it seems wrong to download many extensions you don't need.

**Comment by sveneisenschmidt** [comment]
I don't have time to maintain Silex-Extensions any more - would anyone like to take it over? So you're free to split it up ins everal repositories.

**Pull request closed without merging**